src/disko/mode_generate.py

```diff
@@ -0,0 +1,75 @@
+import json
+import re
+from disko_lib.eval_config import eval_config_dict_as_json
+from disko_lib.logging import info
+from disko_lib.result import DiskoPartialSuccess, DiskoResult, DiskoError
+from disko_lib.generate_config import generate_config
+from disko_lib.run_cmd import run
+from disko_lib.json_types import JsonDict
+
+DEFAULT_CONFIG_FILE = "disko-config.nix"
+
+HEADER_COMMENT = """
```

Comment:
Add a backslash if you don't want an empty first line.

```suggestion
HEADER_COMMENT = """\
```